docs/development.rst

```diff
@@ -0,0 +1,283 @@
+
+        $ make develop
+
+6. This project uses Make to do things in the currently active Python
+   environment. The command:
+
+   .. code-block:: bash
+
+        $ make
+
+   displays a list of valid Make targets and a short description of what each
+   target does.
+
+.. _virtual Python environment: https://docs.python-guide.org/en/latest/dev/virtualenvs/
+
+
+.. _`Manual setup on Windows`:
+
+Manual setup on Windows
+^^^^^^^^^^^^^^^^^^^^^^^
+
+For development of pywbemtools, you may use one of the Unix-like
+environments on Windows (such as CygWin, MinGW, Babun, or Gow) or native
+windows. The pywbemtools tests that run on the Appveyor CI use both CygWin
+and native windows.
+
+Note that Unix-like environments on Windows bring their own Python, so double
+check that the active Python environment is the one you want to install to.
+
+
+.. _`Prerequisite operating system packages for development`:
+
+Prerequisite operating system packages for development
+------------------------------------------------------
+
+The following table lists the prerequisite operating system packages along with
+their version requirements for development of pywbemtools, for the supported
+operating systems and Linux distributions.
+
+The prerequisite operating system packages for installing and running pywbem
+are also needed for development, and can be found in section
+:ref:`Prerequisite operating system packages for install`.
+
+This section is just for information. These packages will be installed as part
+of the steps described in :ref:`Setting up the development environment`.
+
++--------------------------+--------------------+----------------------+-------+
+| Op.system / Distribution | Package name       | Version requirements | Notes |
++==========================+====================+======================+=======+
+| Linux RedHat family      |                    |                      |       |
+| (RHEL, CentOS, Fedora)   |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Linux Debian family      |                    |                      |       |
+| (Ubuntu, Debian,         |                    |                      |       |
+| LinuxMint)               |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Linux SUSE family        |                    |                      |       |
+| (SLES, openSUSE)         |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| OS-X                     |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Windows                  |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+
+
```

Comment:
From here on down to the end of this file, the package names `pywbem` and `pywbem_mock` are used, when tehy should have been changed to `pywbemtools`. Also in URLs.

Reply:
Done. Gone